tests/Sdk/Task/SprykRunTask.yaml

```diff
@@ -0,0 +1,44 @@
+# Copied from Extension/Resources/config/task/SprykRunTask.yaml and changed type to `TYPE_LOCAL_CLI`,
+# because CI doesn't support `TYPE_LOCAL_CLI_INTERACTIVE` type.
+---
+id: "spryk:run:local"
```

Comment:
What is the idea of this test? as I see you don't test from an application. you test task from test env

Reply:
Fixed